tests/rptest/services/redpanda.py

```diff
@@ -1966,6 +1983,7 @@ def stop_node(self, node, timeout=None, forced=False):
                 f"Redpanda node {node.account.hostname} failed to stop in {timeout} seconds"
             )
         except TimeoutError:
+            self._set_trace_loggers_and_sleep(node)
```

Comment:
Would be good to log a message before enabling trace, so that when reading the log the sleep doesn't look like some kind of weird test hang.

Reply:
Done